src/SFA.DAS.ApplyService.Application.UnitTests/Services/AuditServiceTests.cs

```diff
@@ -0,0 +1,222 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoFixture;
+using Moq;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using SFA.DAS.ApplyService.Application.Apply;
+using SFA.DAS.ApplyService.Application.Interfaces;
+using SFA.DAS.ApplyService.Application.Services;
+using SFA.DAS.ApplyService.Domain.Audit;
+
+namespace SFA.DAS.ApplyService.Application.UnitTests.Services
+{
+    [TestFixture]
+    public class AuditServiceTests
+    {
+        private AuditServiceTestsFixture _fixture;
```

Comment:
I'm not fully clued up with AutoFixture but a lot of these tests looks like they could be covered with Moq which is used throughout this code base